xbmc/platform/android/jni/AudioFormat.cpp

```diff
@@ -34,6 +34,7 @@ int CJNIAudioFormat::ENCODING_DTS_HD                   = 0x00000008;
 int CJNIAudioFormat::ENCODING_DOLBY_TRUEHD             = 0x00000009;
 int CJNIAudioFormat::ENCODING_IEC61937                 = 0x0000000d;
 
+int CJNIAudioFormat::CHANNEL_OUT_MONO                  = 0x00000004;
 int CJNIAudioFormat::CHANNEL_OUT_STEREO                = 0x0000000c;
```

Comment:
not enough 👍 

need below like how CHANNEL_OUT_STEREO is handled

      CJNIAudioFormat::CHANNEL_OUT_MONO  = get_static_field<int>(c, "CHANNEL_OUT_MONO ");